target/linux/layerscape/patches-5.4/820-usb-0009-usb-dwc3-Add-workaround-for-host-mode-VBUS-glitch-wh.patch

```diff
@@ -44,7 +44,22 @@ Reviewed-by: Peter Chen <peter.chen@nxp.com>
  
 --- a/drivers/usb/dwc3/core.h
```

Comment:
Please add a fixes: tag to the commit, this probably broke in one of the kernel updates. The build bots are also broken: https://buildbot.openwrt.org/openwrt-21.02/images/#/grid

Reply:
@hauke found the culprit... it was dropped in a kernel bump for some reason....